genesyscloud/external_contacts/resource_genesyscloud_externalcontacts_contact_utils.go

```diff
@@ -89,7 +90,8 @@ func flattenPhoneNumber(phonenumber *platformclientv2.Phonenumber) []interface{}
 	resourcedata.SetMapValueIfNotNil(phonenumberInterface, "display", phonenumber.Display)
 	resourcedata.SetMapValueIfNotNil(phonenumberInterface, "extension", phonenumber.Extension)
 	resourcedata.SetMapValueIfNotNil(phonenumberInterface, "accepts_sms", phonenumber.AcceptsSMS)
-	resourcedata.SetMapValueIfNotNil(phonenumberInterface, "e164", phonenumber.E164)
+	phoneNumber := util.FormatAsCalculatedE164Number(*phonenumber.E164)
```

Comment:
Same here. The function `SetMapValueIfNotNil` would have been checking if `phonenumber.E164` was nil before dereferencing